scripts/linters/python_linter_test.py

```diff
@@ -69,8 +70,12 @@ def test_valid_job_imports(self) -> None:
             os.path.join(os.getcwd(), 'core', 'jobs', 'batch_jobs')
         )
         lint_task_report = (
-            python_linter.check_jobs_imports(batch_jobs_dir, VALID_PY_FILEPATH)
+            python_linter.check_jobs_imports(
+                batch_jobs_dir, VALID_PY_JOBS_FILEPATH)
         )
+        print('DEBUGGING')
+        print(lint_task_report.get_report())
```

Comment:
Should this be here? (If so, should the print statements above/below this line be more descriptive?)

Reply:
nope--removed